os/pm/pm_metrics.c

```diff
@@ -330,7 +331,7 @@ int pm_metrics(int milliseconds)
 		pm_resumed = pm_resume(PM_IDLE_DOMAIN);
 	} else {
 		pm_resumed = -1;
-		pmdbg("Unable to resume IDLE Domain\n");
+		pmdbg("%s: Resume IDLE Domain\n", clog_message_str[CMN_LOG_FAILED_OP]);
```

Comment:
check the correct usage according to user. Which is giving the right information to the user? update the usage accordingly.  
1. Operation Failure: Resume IDLE Domain 
2. Resume Idle domain operation failure

Reply:
ok